UPDATE-README

```diff
@@ -3,3 +3,7 @@
 
 be925ea: Add mumble.bypassCore to your config file. True means mumble will fall back to the local database if the
     connection to Core fails.
+
+d2d6ae4: Moved startAuthenticator.sh to bcs-mumble-auth.sh. Expects default install path of /opt/bcs/bcs-mumble.
```

Comment:
Since the main supported platform for core is currently Debian (7,8), lets see if we can either remove these default paths all together or use a debian standard if one exists. Also these apps should be installed in virtualenv's, so we should prolly expect that we will have valid path resolution to the ~/bin folder out of the box and users should only be able to run sh/bin files while having the virtualenv activated.